### PR TITLE
NextJS guide fixes

### PR DESCRIPTION
**Pull request by perkinsjr**
1. Better explanation of what to expect when navigating to the URL 
2. New video on the init page.
3. Added Diffs
4. Changed to primitive queries
5. Added a remove imports to stop errors.
6. Added some more "restart server" for plugins

### CODE REVIEW

**Comment by perkinsjr** [comment]
Fixes #965 #963

File: content/guides/tina-cloud/getting-started/defining-content.md [hunk 1]
```diff
@@ -49,31 +46,44 @@ As you can see, we have a quite a few fields that we want our content team to be
...
+      label: "Blog Post Body",
+      name: "body",
+      isBody: true,
+    },
   ]
 }
 ```
 
-Here we have created a label with a human friendly name "Blog Posts", the queryable name will be "posts", and all content should be saved in the "\_posts" directory, which is where the NextJS starter blog post content lives. You can see we just havea a single field called `title` which is of type `string`.
+We have only replaced a single line so far, which is to replace the `path` to the where the content from the Starter Blog is stored.
```
Comment (jeffsee55) [inline content/guides/tina-cloud/getting-started/defining-content.md:73]:
```suggestion
We have only replaced a single line so far, which is to update the `path` to the where the content from the Starter Blog is stored. 
```

File: content/guides/tina-cloud/getting-started/markdown-plugin.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ The `ui` property allows you to control the output of the GraphQL-generated form
 
 ### A quick test
 
-If you check the Next Starter now and enter edit mode, you will see the body now has Markdown options to make editing easier:
+Restart your application by shutting it down and running `yarn tina-dev` and enter edit mode, you will see the body now has Markdown options to make editing easier:
```
Comment (jeffsee55) [inline content/guides/tina-cloud/getting-started/markdown-plugin.md:86]:
```suggestion
Restart your application by shutting it down and running `yarn tina-dev`. Be sure you're in edit mode and you will see the body now has Markdown options to make editing easier:
```

Just thinking they'll be in edit mode already from the previous steps, but if they're not they no how to get there

File: content/guides/tina-cloud/getting-started/next-changes.md [hunk 3]
```diff
@@ -282,17 +310,24 @@ export default function Post({ data, slug }) {
             <article className="mb-32">
               <Head>
                 <title>
-                  {title} | Next.js Blog Example with {CMS_NAME}
+-                {post.title} | Next.js Blog Example with {CMS_NAME}
++                {title} | Next.js Blog Example with {CMS_NAME}
```
Comment (jeffsee55) [inline content/guides/tina-cloud/getting-started/next-changes.md:314]:
```suggestion
```

File: content/guides/tina-cloud/getting-started/defining-content.md [hunk 4]
```diff
@@ -49,31 +46,44 @@ As you can see, we have a quite a few fields that we want our content team to be
 
 ### Making changes to the Schema
 
-Open up the Tina `schema.ts` file located at `/.tina/schema.ts` To begin with underneath the object we provided, we need to create a new collection object:
+Open up the Tina `schema.ts` file located at `/.tina/schema.ts` To begin with underneath the object we provided, we need to replace the current collection with the content we want:
 
 ```js
```
Comment (logan-anderson) [inline content/guides/tina-cloud/getting-started/defining-content.md:51]: Should this be diff and not JS?

Comment (perkinsjr) [reply content/guides/tina-cloud/getting-started/defining-content.md:51]: Yup should be diff. I did so many changes forgot to go back through!

**Review by jeffsee55 — APPROVED** [review]

**Pull request merged**